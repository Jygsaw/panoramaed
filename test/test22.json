{
  "teachers": [
    "Ra's al Ghul",
    "Uncle Ben"
  ],
  "students": [
    "Bruce Wayne",
    "Peter Parker"
  ]
}
