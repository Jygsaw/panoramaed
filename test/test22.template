<html>
  <head>
    <title>Title</title>
  </head>
  <body>
    <h1>Header</h1>
    <* EACH teachers teacher *>
      <* teacher *>
    <* ENDEACH *>

    <* EACH students student *>
      <* student *>
    <* ENDEACH *>
  </body>
</html>
