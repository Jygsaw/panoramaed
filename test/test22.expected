<html>
  <head>
    <title>Title</title>
  </head>
  <body>
    <h1>Header</h1>
      Ra's al Ghul
      Uncle Ben

      Bruce Wayne
      Peter Parker
  </body>
</html>
